packages/web-components/src/components/va-table/va-table-inner/va-table-inner.scss

```diff
@@ -47,11 +47,19 @@
   }
 }
 
+:host .usa-table th[data-sortable="false"] {
+  padding-right: 0;
+}
+
 ::slotted([slot='headers']) {
   background: var(--vads-color-base-lightest);
   font-weight: 700;
 }
 
+:host .text-align-right {
```

Comment:
Could we use the [USWDS alignment class](
https://designsystem.digital.gov/utilities/paragraph-styles/#text-align)? Or maybe better to use a [vads utility class](https://design.va.gov/foundation/utilities/text-align)?

Reply:
That seems doable, yep! I used the text-align-right class because there was already one defined in the va-table.scss file, but you're right that it would be even better to use the vads utility class.